plugins/lookup/spec.py

```diff
@@ -18,15 +18,17 @@
         structure of yang document. The tree structure document is as per RFC 8340 which helps to consume
         the yang document along with json and xml configuration skeleton.
     options:
-      _terms:
```

Comment:
This change doesn't seem correct. Lookup plugins mandatory parameter is accepted as `_terms`

Reply:
Updated